Kernel/Arch/aarch64/Dummy.cpp

```diff
@@ -28,38 +29,16 @@ namespace Kernel::PCI {
 bool g_pci_access_io_probe_failed { false };
 bool g_pci_access_is_disabled_from_commandline { true };
 
+void initialize()
+{
+    dbgln("PCI: FIXME: Enable PCI for aarch64 platforms.");
+    g_pci_access_io_probe_failed = true;
 }
 
-// x86 init
-
-multiboot_module_entry_t multiboot_copy_boot_modules_array[16];
-size_t multiboot_copy_boot_modules_count;
-
-extern "C" {
-READONLY_AFTER_INIT PhysicalAddress start_of_prekernel_image;
-READONLY_AFTER_INIT PhysicalAddress end_of_prekernel_image;
-READONLY_AFTER_INIT size_t physical_to_virtual_offset;
-// READONLY_AFTER_INIT FlatPtr kernel_mapping_base;
-READONLY_AFTER_INIT FlatPtr kernel_load_base;
-READONLY_AFTER_INIT PhysicalAddress boot_pml4t;
-READONLY_AFTER_INIT PhysicalAddress boot_pdpt;
-READONLY_AFTER_INIT PhysicalAddress boot_pd0;
-READONLY_AFTER_INIT PhysicalAddress boot_pd_kernel;
-READONLY_AFTER_INIT Kernel::Memory::PageTableEntry* boot_pd_kernel_pt1023;
-READONLY_AFTER_INIT char const* kernel_cmdline;
-READONLY_AFTER_INIT u32 multiboot_flags;
-READONLY_AFTER_INIT multiboot_memory_map_t* multiboot_memory_map;
-READONLY_AFTER_INIT size_t multiboot_memory_map_count;
-READONLY_AFTER_INIT multiboot_module_entry_t* multiboot_modules;
-READONLY_AFTER_INIT size_t multiboot_modules_count;
-READONLY_AFTER_INIT PhysicalAddress multiboot_framebuffer_addr;
-READONLY_AFTER_INIT u32 multiboot_framebuffer_pitch;
-READONLY_AFTER_INIT u32 multiboot_framebuffer_width;
-READONLY_AFTER_INIT u32 multiboot_framebuffer_height;
-READONLY_AFTER_INIT u8 multiboot_framebuffer_bpp;
-READONLY_AFTER_INIT u8 multiboot_framebuffer_type;
 }
 
-extern "C" {
-FlatPtr kernel_mapping_base;
+// kprintf.cpp
+void set_serial_debug_enabled(bool)
+{
+    dbgln("FIXME: Add support for serial debugging");
```

Comment:
`"FIXME: Add support for changing state of serial debugging"` ?

We already support serial debug, no? we just can't toggle it.

Reply:
Right, you are correct. I changed the wording of the FIXME.